apps/resources/admin.py

```diff
@@ -14,7 +14,7 @@ def get_formset(self, request, obj=None, **kwargs):
         """
         formset = super().get_formset(request, obj, **kwargs)
         resource_field = formset.form.base_fields['resource']
-        resource_field.queryset = resource_field.queryset.order_by('title')
+        resource_field.queryset = resource_field.queryset.filter(categories=obj).order_by('title')
```

Comment:
Is this has been requested? Sorry, but they might want to feature resource from the different category, if it has been requested then it's fine otherwise leave as it was.

Reply:
OK, I reverted that now. Seems too contentious. :-)